Python/Product/PythonTools/visualstudio_py_debugger.py

```diff
@@ -2318,7 +2318,7 @@ def detach_threads():
 
 def new_thread(tid = None, set_break = False, frame = None):
     # called during attach w/ a thread ID provided.
-    if tid == debugger_thread_id:
+    if tid and tid == debugger_thread_id:
```

Comment:
`None == <anything except None>` should always return `False`, so there shouldn't be a need to check that `tid` is not None first. I'm wondering whether Jython gets this wrong somehow, or if its implementation of `thread.start_new_thread()` does not return an identifier?